tests/test_visitors/test_tokenize/test_primitives/test_string_tokens/test_string_modifier.py

```diff
@@ -43,6 +41,36 @@ def test_correct_prefix(
     assert_errors(visitor, [])
 
 
+@pytest.mark.parametrize('modifier', [
+    'r',
+    'rb',
+    'fr',
+])
+@pytest.mark.parametrize('primitive', [
+    '{0}"\n"',
+    "{0}'\n'",
+    '{0}"Big \ntext"',
+    "{0}'Format \n123'",
+])
+def test_correct_prefix_raw_string(
```

Comment:
We need `test_wrong_prefix_raw_string` case as well.

Reply:
See f3b55429539657bced0dd0d376d1efde116ee957